homeassistant/helpers/event.py

```diff
@@ -33,6 +25,7 @@
 from homeassistant.core import (
     CALLBACK_TYPE,
     Event,
+    EventStateChangedData as EventStateChangedData,  # noqa: PLC0414
```

Comment:
Moved to `core` to prevent circular imports.
If this PR is accepted, I can update the imports in unrelated files.